<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserModifyDialog</class>
 <widget class="QDialog" name="UserModifyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入旧密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OldPasswdlineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewPasswdlineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ConfirmPasswdlineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请再次输入新密码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKpushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>480</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserNamelineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户等级：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="levelcomboBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>UserNamelineEdit</tabstop>
  <tabstop>OldPasswdlineEdit</tabstop>
  <tabstop>NewPasswdlineEdit</tabstop>
  <tabstop>ConfirmPasswdlineEdit</tabstop>
  <tabstop>levelcomboBox</tabstop>
  <tabstop>OKpushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
